src/screens/HomeScreen.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import {View, Image, Text, Dimensions, StyleSheet, SafeAreaView, Button, TouchableOpacity} from 'react-native';
+import {useNavigation} from '@react-navigation/native';
+
+const windowWidth = Dimensions.get('window').width;
+const windowHeight = Dimensions.get('window').height;
+export default function HomeScreen() {
+    const navigation = useNavigation();
+    return (
+        <SafeAreaView>
+            <Image source={require('../assets/images/splash_logo.png')} style={styles.ImageStyle} resizeMode='contain'/>
```

Comment:
Can we just use import instead of a require? @usaidpeerzada

Reply:
In the rn docs I found two ways to get an image, if it is locally downloaded then require , if we get it from some url then uri:

I will check if there is a way to import.